packages/web/src/js/FileUploader.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-console */
```

Comment:
I prefer disable-next-line as it is more strict. What do you think?

Reply:
I'd say: safer to use. Because this rule (or any rule) should be turned back on as soon as possible. Now it's off for the whole file even though there are just 3 places where we actually need it to be turned off.

I'd prefer to use `eslint-disable-next-line` along with a reason.

From [ESlint docs](https://eslint.org/docs/latest/use/configure/rules#disabling-rules):

> - **Use with Caution.** Disabling ESLint rules inline should be restricted and used only in situations with a clear and valid reason for doing so. Disabling rules inline should not be the default solution to resolve linting errors.
> - **Document the Reason.** Provide a comment explaining the reason for disabling a particular rule after the -- section of the comment. This documentation should clarify why the rule is being disabled and why it is necessary in that specific situation.

Unfortunately it doesn't seem there is a way to strictly require the reason to be present…

Reply:
Ok, never mind, I have solved this by refactoring to warning utility. Please see https://github.com/lmc-eu/spirit-design-system/pull/1188 first.

Reply:
Nice! Please just always keep the reason right after the disabling statement. You will say thank you to your future self 🙂.

Reply:
I know that is why I rather refactor this. Because it was simpler than keeping a reason in this case. It was too much places where I must keep it ;-)